### PR TITLE
libwebp: more targets

### PR DESCRIPTION
**Pull request by pdknsk**
After the targets have run for a few days without problems, everything moves to the `libwebp` repo.

### CODE REVIEW

**Comment by inferno-chromium** [comment]
@jzern - can you please take a first pass review of this. Feel others to add other libwebp devs as needed.

**Comment by jzern** [comment]
Oh those are global approvals, my mistake. I'll just lgtm the individual commits as I got through.

File: projects/libwebp/fuzz_demux_api.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    WebPChunkIterator chunk_iter;
+    if (WebPDemuxGetChunk(demux, "EXIF", 1, &chunk_iter))
+      WebPDemuxNextChunk(&chunk_iter);
+    WebPDemuxReleaseChunkIterator(&chunk_iter);
+    if (WebPDemuxGetChunk(demux, "ICCP", 0, &chunk_iter)) // 0 == last
+      WebPDemuxPrevChunk(&chunk_iter);
+    WebPDemuxReleaseChunkIterator(&chunk_iter);
+    // Skips FUZZ because the Demux API has no concept of (un)known chunks.
```
Comment (jzern) [inline projects/libwebp/fuzz_demux_api.c]: you can ask for an arbitrary chunk fourcc

**Comment by jzern** [comment]
lgtm for all the changes now. Thanks!

File: projects/libwebp/fuzz_demux_api.c (hunk 1 above)
Comment (pdknsk) [reply projects/libwebp/fuzz_demux_api.c]: I meant that the *Demux API* is oblivious to chunk names, whereas the *Mux API* takes a different path for unknown chunks. So there is no additional coverage asking for `FUZZ` here.

File: projects/libwebp/Dockerfile [hunk 2]
```diff
@@ -20,5 +20,8 @@ RUN apt-get update && apt-get install -y autoconf make libtool
 RUN git clone https://chromium.googlesource.com/webm/libwebp
 ADD http://cdn.pwmon.org/oss-fuzz/libwebp/fuzz_seed_corpus.zip $SRC/
 COPY build.sh fuzz.h fuzz.dict $SRC/
-COPY fuzz_simple_api.c fuzz_simple_api.options $SRC/
+COPY fuzz_simple_api.c $SRC/
+COPY fuzz_advanced_api.c $SRC/
+COPY fuzz_animation_api.c $SRC/
+COPY fuzz_demux_api.c $SRC/
```
Comment (jonathanmetzman) [inline projects/libwebp/Dockerfile]: nit: Why not *.c?

Comment (jonathanmetzman) [reply projects/libwebp/Dockerfile]: I suppose it isn't important to fix this if the targets will be moved upstream soon.

**Pull request merged**